lbuild/__main__.py

```diff
@@ -243,7 +243,9 @@ def perform(self, args, parser, config, repo_options):
 
         config.selected_modules.extend(args.modules)
         build_modules, module_options = get_modules(parser, repo_options, config.options, config.selected_modules)
-        parser.build_modules(args.path, build_modules, repo_options, module_options, log)
+        success = parser.build_modules(args.path, build_modules, repo_options, module_options, log)
+        if not success:
+            exit(1)
```

Comment:
This is ugly, but I didn't want to research how this function is called, so you change it ;-P